docker-compose.yml

```diff
@@ -54,8 +54,8 @@ services:
     networks:
       - lychee
     environment:
-      #- PUID=1000
-      #- PGID=1000
+      - PUID=${PUID:-1000}
+      - PGID=${PGID:-1000}
```

Comment:
Same query as with `APP_URL` in the other discussion.

Reply:
I am trying to look this one up.
https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/
It is unclear to me whether the .env file is fully path through or if it is set for the docker compose which then use the environment definition... :|

Reply:
https://stackoverflow.com/a/74544228

Reply:
I am indeed maybe a bit confused. Seems like indeed there is some difference behaviors...

Reply:
Hmm.. So we probably either need to enable more of those in docker-compose.yml or set .env as an `env_file`?

Reply:
Or use:
```
    env_file:
      - path: ./.env
        required: false
```

Reply:
Yup, that looks good to me

Reply:
See the Harden PR :)